### ISSUE TITLE
Issue "KeyError: key not found: "data"" when trying to set a different config for schema_path

### ISSUE THREAD

**Issue by victorfeijo** [yuki24/artemis#83]
Hey @yuki24 ! First of all thanks for putting time and effort to build this lib. I've been struggling for a while to develop something on top of `graphql-client` and so far just got frustrated. Ended up checking out Artemis in some other random Github issue and found it to be very helpful, indeed makes you much more productive. In 5 minutes I was able to get my queries to work and that was such a great experience!

However, now I'm having an issue when trying to change the `schema_path`. I was able to run the rake task to generate a schema dump and the file was saved at the following location: `vendor/graphql/schema/shopify_partners.json`. With no changes to the default `config/graphql.yml` file, I'm able to generate and perform my queries without any hassle. But when I change the `schema_path` to be something else for example (and also move the generated file from `vendor/` to the new path):

```yml
  schema_path: "db/shopify_graphql_schemas/shopify_partners.json"
```

I start to get the following error when operating any queries:

```ruby
> ShopifyPartners.transactions
KeyError: key not found: "data"
from /Users/victorfeijo/.rvm/gems/ruby-3.1.1/gems/graphql-1.13.11/lib/graphql/schema/loader.rb:17:in `fetch'
```

Found it to be super weird because If I set my schema path to the original path:

```yml
  schema_path: "vendor/graphql/schema/shopify_partners.json"
```

It actually works! So seems like anything other than the original generated path won't work 😅 . I'm fine with the schema living under the `vendor/` path, but from what I've checked from the lib code it was supposed to work with a custom path. Just wanna make sure I didn't do anything wrong and I'm eventually not loading the schema properly. Does that make sense? Let me know if you need any other info from my side.

Thanks!

**Comment by yuki24** [comment]
@victorfeijo Thanks for reporting this bug! This seems unexpected. Just looking at the `lib/artemis/railtie.rb`, it seems to forcefully be adding a file name:

https://github.com/yuki24/artemis/blob/23a43cab0b2e701035823e4de51b8d55e9bbfcd7/lib/artemis/railtie.rb#L53

So if you remove the file name from the path:

```yaml
  schema_path: "db/shopify_graphql_schemas"
```

Though the expacted behaivour here is that the full path should work without problems. I'll look into fixing this later. Thanks!

**Comment by victorfeijo** [comment]
hey, @yuki24 thanks for the quick reply! I've tried what you've suggested and it didn't work as well. However, I've decided to restart all my rails processes (killed `spring` as well) and it seems to be working now with a custom path 😅 . This is how I've managed to do it:

<img width="782" alt="Screen Shot 2022-04-22 at 11 12 21" src="https://user-images.githubusercontent.com/8865899/164589248-66f0d0b2-5271-4223-8b85-57a1bff842a2.png">

I'm sorry for pointing out an issue that actually works, I'll assume it was some cache between `rails console` sessions. I'm gonna be closing the issue because the `schema_path` is working perfectly! Sorry and thanks again for the quick response :)
